src/ode_adaptive.cc

```diff
@@ -119,6 +119,7 @@ void OdeAdaptive::solve_adaptive (double tint, double dt0, const char *dirout, i
     //extra completion things (to be overridden in derived class)
     after_solve();
 
+    delete[] solout;
```

Comment:
Thanks for catching this. The `solout` variable is an array of vectors, so the inner vectors also have to be freed.

Reply:
You're welcome. The `solout` is created by a new [] operator and would be freed by delete []. I think the inner vectors, still objects, will be freed properly by their destructors. It is an array of vector objects but not array of `double *`. Thus no change is needed.